tests/unit/ml_handlers/test_openai.py

```diff
@@ -69,6 +68,21 @@ def test_invalid_openai_name_parameter(self):
         with pytest.raises(Exception):
             self.wait_predictor("proj", "test_openai_nonexistant_model")
 
+    def test_unknown_arguments(self):
+        self.run_sql("create database proj")
+        with pytest.raises(Exception):
+            self.run_sql(
+                f"""
+                create model proj.test_openai_unknown_arguments
+                predict answer
+                using
+                    engine='openai',
+                    question_column='question',
+                    openai_api_key='{OPEN_AI_API_KEY}',
+                    model='unknown_argument';  --- this is a wrong argument name
```

Comment:
Minor nitpick: maybe rename the argument to something that has less probability of being a legit one in the future, e.g. `evidently_wrong_argument`